proto/tikvpb.proto

```diff
@@ -70,7 +71,7 @@ service Tikv {
 message BatchCommandsRequest {
```

Comment:
I think it's better to add it to `BatchCommands`, so that it can be batched with other requests to save RPC.